docs/known_limitations.md

```diff
@@ -0,0 +1,9 @@
+# Known Limitations
+
+Despite the list of enabled APIs is documented, there might be some cases where it's not clear whether a certain functionality works. This is a list of known limitations within MicroShift:
+
+## Idling/Unidling Applications
+
+The `oc` client provides a way to idle applications in order to reduce power consumption. By adding specific annotations, `oc` will scale all workloads referenced to a service down to zero. This is a fully working feature in MicroShift.
+
+However, when the application service/route receives network traffic, OVN-Kubernetes will detect it and write an annotation to scale that application back up. MicroShift does not provide the unidling controller though, so this functionality won't work.
```

Comment:
OVN-Kubernetes sends an Normal type event to scale ...

Reply:
https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/ovn/controller/unidling/unidle.go#L207

Reply:
Good catch, idling is annotation based, so wrong assumption from my side.